src/compiler/checker.ts

```diff
@@ -12997,7 +12997,7 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
                 // conflicting flags.

```

Comment:
please update this comment to read "Report an error if there's a symbol declaration with the same name and conflicting flags.", since there's no longer an error for any earlySymbol with the same name.

Reply:
Updated 🙂